Source/JavaScriptCore/b3/B3ReduceStrength.cpp

```diff
@@ -2181,6 +2181,7 @@ class ReduceStrength {
                     m_changed = true;
                 }
             }
+            UNUSED_PARAM(address); // placate clang static analyzer.
```

Comment:
I think `UNUSED_VARIABLE()` would be more appropriate here since it's not a function parameter:

```suggestion
            UNUSED_VARIABLE(address); // placate clang static analyzer.
```

Reply:
If I'm not mistaken, the project guidelines explain that the comments should not be redundant.
Everybody understands that UNUSED_VARIABLE(address) is added to placate a tool, you don't need to add a comment due to that.

The obvious fix is to fix the source, e.g. to remove the redundant assignment.
We did not want to do that and added the override. The comment should probably explain why the obvious course was not taken, rather this is a better solution.